horace_core/sqw/@sqw/private/resolution_plot_private.m

```diff
@@ -16,26 +16,27 @@ function resolution_plot_private (x0,C,iax,flip)
 %          ellipse section is drawn at a positive value along that axis
 %
 %   flip    If true, flip the plot axes; if false, not
-%           For plotting if display axes are reversed from plot axes.
+%           For plotting if the display axes are reversed from plot axes.
 
 
 % Original author: T.G.Perring
-%
-% $Revision:: 1759 ($Date:: 2020-02-10 16:06:00 +0000 (Mon, 10 Feb 2020) $)
 
 
 % Check input arguments
 if ~(isnumeric(x0) && numel(x0)==2 && all(isfinite(x0)))
```

Comment:
Could be replaced by:
```matlab
validateattributes(x0, {'numeric'}, {'vector', 'numel', 2, 'finite'}, 'resolution_plot_private', 'x0')
validateattributes(C, {'numeric'}, {'size', [4,4]}, 'resolution_plot_private', 'C')
```
Can't cleanly replace the third check due to variable size and isunique.